File: app/src/main/java/com/apkide/ui/preferences/EditorPreferencesFragment.java
package com.apkide.ui.preferences;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import com.apkide.ui.R;

public class EditorPreferencesFragment extends PreferenceFragmentCompat {
	@Override
	public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
		addPreferencesFromResource(R.xml.editor_preferences);
	}
}

File: app/src/main/java/com/apkide/ui/preferences/EditorPreferencesFragment.kt
package com.apkide.ui.preferences

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.apkide.ui.R

class EditorPreferencesFragment : PreferenceFragmentCompat() {
	override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
		addPreferencesFromResource(R.xml.editor_preferences)
	}
}